tests/stdlib/test_math.py

```diff
@@ -8,4 +9,46 @@ def test_sqrt(self):
         self.assertCodeExecution("""
             import math
             print(math.sqrt(1))
-            """)
+        """)
+
+    def test_exp_standard(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.exp(0))
+            print(math.exp(-1))
+            print(math.exp(1))
+            print(math.exp(0.0001))
+            print(math.exp(-0.0001))
+            print(math.exp(-100000))
+        """)
+
+    @expectedFailure
+    def test_exp_fail_empty(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.exp())
+        """)
+
+    @expectedFailure
+    def test_exp_fail_nan(self):
+        self.assertCodeExecution("""
+            print(math.exp('this is not a number'))
+        """)
+
+    @expectedFailure
+    def test_exp_fail_two_args(self):
+        self.assertCodeExecution("""
+            print(math.exp(1, 2))
+        """)
+
+    @expectedFailure
+    def test_exp_fail_array_arg(self):
+        self.assertCodeExecution("""
+            print(math.exp([1, 2, 3]))
+        """)
+
+    @expectedFailure
+    def test_exp_fail_large(self):
+        self.assertCodeExecution("""
+            print(math.exp(100000))
+        """)
```

Comment:
Coding convention for this project is to have a newline at the end of the file. Please add a newline.